yugabyte/yugabyte-db
# [#6373] Platform: Pause/Resume universe.

**Description**:
Ideally, there would be a way to pause the instances associated with a universe. This would be particularly useful for clusters that have some data loaded for tpcc, where the data load is time-consuming/expensive, but the cluster itself is only in use during periodic workload execution.

**Testing**: 

1. Create a universe (AWS) 
2. Go to the universe -> actions -> pause universe
3. It should be paused after some time and the status should be updated.
4. One can resume the universe from the same place.

Expectations: 
Everything should be working after the successful resume.
The user should not be able to perform any tasks except delete.

**sbt test**: 
```
... 15 common frames omitted
[info] Passed: Total 1145, Failed 0, Errors 0, Passed 1141, Skipped 4
[success] Total time: 726 s, completed 2 Feb, 2021 11:54:11 AM
```
